index.md

```diff
@@ -1,19 +1,81 @@
+"collapse" can be a change in API that is not backwards compatible, a regression
+in a tool whose version was not pinned, a change in URL of an external resource,
+etc.
 
-Life Monitor exposes its functionalities through a RESTful API defined with
-[OpenAPI](https://swagger.io/specification). Below you'll find links to the
-current version of the API, which corresponds to the `master` branch in the
-GitHub repository.
+Life Monitor aims to facilitate the creation, sharing, periodic execution and
+monitoring of workflow tests, ensuring that software collapse is detected and
+communicated to the authors, in the hope it will be solved thus extending the
+useful life of the workflows.
 
-* Formatted with RapiDoc:  [lm-openapi-rapidoc.html](./lm-openapi-rapidoc.html)
-* Static page automatically created with openapi-generator: [lm-openapi-html](./lm-openapi-html/index.html)
 
-You can find the raw YAML API specifications in [the source code repository](https://github.com/crs4/life_monitor/tree/master/specs).  
+## Goals
+
+* Provide a central aggregation point for your workflow test statuses and outputs
+  from various testing bots (e.g., TravisCI, GitHub Actions, your own
+  Jenkins instance, etc.).
+* Integrated Jenkins-based workflow test execution service.
+* Facilitate periodic automated execution of tests for Galaxy, Nextflow and CWL
+  workflows.
+* Web interface, CLI client, REST API
+
+
+## How to use it
+
+At the moment, the Life Monitor can be used through its [REST
+API](lm_api_specs).  **Follow the link** for [documentation and
+specifications](lm_api_specs).
+
+Integration with the [Workflow Hub](https://workflowhub.eu/) is in the works, as
+well as a web interface and a command line client.
+
+A critical component to define and exchange workflow tests with the Life Monitor
+is the [Workflow RO-crate](https://about.workflowhub.eu/Workflow-RO-Crate/) and
+[Workflow RO-crate testing extension](https://github.com/crs4/life_monitor/wiki/Workflow-Testing-RO-Crate).
+
+
+## Road map
+
+Life Monitor is still in early development.  Here is our planned development road map.
+
+
+#### End of 2020
+- [x] Support for receiving workflow POSTs as Workflow RO-crate;
+- [x] Relatively stable interface and implementation for test outcome retrieval;
+- [x] Complete first version of Workflow RO-crate draft test specification;
+- Support monitoring tests running on external testing services (`test.instance` within the WRO-crate test specs).
+    - [x] TravisCI
+    - [x] Jenkins
+
+#### Spring 2021
+  - [x] Workflow testing RO-crate template creation (integrated in
+        [ro-crate-py](https://github.com/ResearchObject/ro-crate-py))
+  - [ ] Alpha release on <https://lifemonitor.eu>
+
+#### Mid 2021
+  - [ ] WorkflowHub integration?
+  - [ ] Command line client
+  - [ ] Web interface
+  - Support monitoring tests running on external testing services (`test.instance` within the WRO-crate test specs).
```

Comment:
I'd remove the " (`test.instance` within the WRO-crate test specs)" bit (see the above comment).

Reply:
done